views/account.pug

```diff
@@ -13,5 +13,6 @@ block content
   .button-group
     button(onclick="location.href='/upload'") Publish
     button(onclick="location.href='/threads'") Read
+    button(onclick="location.href='/replies'") Replies
```

Comment:
How do you feel about naming it `My Replies` instead?

Reply:
I like that suggestion. How would you feel about also renaming `Read` to either `Threads` or `My Threads` to make it a bit more clear?